File: src/main/kotlin/com/itmo/microservices/demo/order/api/dto/OrderItemDto.java
package com.itmo.microservices.demo.order.api.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.UUID;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class OrderItemDto {
    private UUID uuid;
    private String title;
    private int price;

    public UUID getUuid() {
        return uuid;
    }

    public String getTitle() {
        return title;
    }

    public int getPrice() {
        return price;
    }
}

File: src/main/kotlin/com/itmo/microservices/demo/order/api/dto/OrderItemDto.kt
package com.itmo.microservices.demo.order.api.dto

import lombok.AllArgsConstructor
import lombok.Builder
import lombok.Data
import lombok.NoArgsConstructor
import java.util.*

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
class OrderItemDto(uuid: UUID?, title: String?, price: Int) {
    val uuid: UUID? = null
    val title: String? = null
    val price: Int = 0
}